src/main/java/staffconnect/model/person/comparators/ModuleComparator.java

```diff
@@ -15,4 +15,9 @@ public class ModuleComparator implements Comparator<Person> {
     public int compare(Person p1, Person p2) {
         return p1.getModule().toString().compareToIgnoreCase(p2.getModule().toString());
     }
+
+    @Override
+    public String toString() {
+        return "Module by alphabetical order";
```

Comment:
Perhaps a better word for this is `alphanumeric` as it contains both alphabets and numbers?

Reply:
oo yes thankyou